tests/image/test_deploy.py

```diff
@@ -174,6 +174,7 @@ def test_progress_bar_logic():
         'Progress:  65%|██████▌   | 65/100'
         'Progress:  75%|███████▌  | 75/100'
         'Progress: 100%|██████████| 100/100'
+        'Progress: 100%|██████████| 100/100'
```

Comment:
The duplication is required because newer version of `tqdm` reprints the progress bar when closing:

![image](https://user-images.githubusercontent.com/163367/104134856-f8c17980-536a-11eb-81a4-a96a6898bae1.png)